inf-terraform-aws/files/Makefile

```diff
@@ -4,7 +4,11 @@ DESCRIPTION              := The '$(NAME)' is a is a prototype for an ODS quickst
 PWD                      := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 GEMS_HOME                ?= $(PWD)/vendor/bundle
 INSTALL_REPORT_HOME      := ./reports/install
-SHELL                    := /usr/bin/env bash
+SHELL                    := /usr/bin/env bash -o pipefail
```

Comment:
`SHELL` might be better be defined in this case as just `/bin/bash`

Reply:
Could you explain that?

Reply:
I assume `/bin/bash` is always found, but not always `/usr/bin/env`

Reply:
and also, initial proposal implies more commands

Reply:
a git grep in ods-quickstarters lists both concepts being used - **/bin/bash** and **/usr/bin/env bash** - what is the correct approach now ?